src/FINDOLOGIC/Helpers/ResponseHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FINDOLOGIC\Helpers;
```

Comment:
The used namespace makes this project look global/one level higher than the other PHP projects. `FINDOLOGIC` should be followed by a project-specific namespace part.

Reply:
Good catch. I do not know why this was like that. I changed it to `FINDOLOGIC\Api`